GUIDE_zh.md

```diff
@@ -2,113 +2,113 @@
+此数据存档过程——将二进制文件先压缩，再编制二维码，最后打包成存档文件——显然会比简单书写人类可读的文本要复杂得多。提取档案的过程，即从二维码到压缩的二进制数据；从压缩到解压；从存档文件到多个文件；从文本文件到人类可读文本，也同样复杂。然而，此复杂过程相比其它方式，可以计算机可读的相对简单方式存储更多数据。
 
 若因为如此复杂而给您造成困难和导致高成本，我们深表歉意。但我们猜测，若真如此，本指南及人类可读的 Tech Tree 将会降低复杂性，而且可能比存档内容更实用，至少当计算机足够先进时，本档案库的数据复杂性会显得易于操作。
 
 ## 文件、目录、仓库和数据格式
 
-我们将讨论本档案库在逻辑上的划分方式，这可能极具指导意义。 我们会特别讨论文件、目录和数据格式，这些可能十分实用。
+我们将讨论本档案库在逻辑上的划分方式，这可能极具指导意义。我们会特别讨论文件、目录和数据格式，这些可能十分实用。
 
-一组数据组合一起，形成符合逻辑的实体并带有单一名称，即成为一个文件。您可以将数据想象成沙子，文件就是仅可用来装沙子的袋子。 目录是文件的集合。您可以将其看作是一种仅可装进其他袋子的袋子。 以此类推，每个仓库包含一个外层目录，称为“根目录”，其中含有一系列文件和/或一系列目录。 每个目录本身又包含文件和目录。
+一组数据组合一起，形成符合逻辑的实体并带有单一名称，即成为一个文件。您可以将数据想象成沙子，文件就是仅可用来装沙子的袋子。目录是文件的集合。您可以将其看作是一种仅可装进其他袋子的袋子。以此类推，每个仓库包含一个外层目录，称为“根目录”，其中含有一系列文件和/或一系列目录。每个目录本身又包含文件和目录。
 
-我们选择此结构的原因在于，将文件分成不同群组比单一的文件集合更易于管理。 外层目录下的某个文件之识别符包含其所有从属目录的名称：从根目录开始，紧接着是其本身的名称，各个名称中间用字符 / 隔开。 例如，在根目录下有一个名为 README.md 的文件，其识别符为 /README.md，而识别符为 /public/www/index.html 的文件即 index.html 文件，位于根目录内“public”目录下的“www”目录中。
+我们选择此结构的原因在于，将文件分成不同群组比单一的文件集合更易于管理。外层目录下的某个文件之识别符包含其所有从属目录的名称：从根目录开始，紧接着是其本身的名称，各个名称中间用字符 / 隔开。例如，在根目录下有一个名为 README.md 的文件，其识别符为 /README.md，而识别符为 /public/www/index.html 的文件即 index.html 文件，位于根目录内“public”目录下的“www”目录中。
 
-每个仓库又含有两个名字，由一个分隔符分开，这个分隔符在本档案库中为“_ ”或称为“下划线字符”。 （此前，曾使用“/”或斜杠，但也用于指示目录，因此为了稍作区别，我们使用“_”。）第一个名字是拥有该仓库的 GitHub 帐户名称；第二个名字则是该单个仓库的名称。 将仓库和文件标识符两相组合，即可用于单独标识本档案库中的单个文件。 例如，在本档案库内的 GitHub 帐户"rezendi"中，有个仓库"ykarma"。其内，目录"web"下的文件"package.json"可单独标识为 rezendi_ykarma 中的 /web/package.json。
+每个仓库又含有两个名字，由一个分隔符分开，这个分隔符在本档案库中为“_”或称为“下划线字符”。（此前，曾使用“/”或斜杠，但也用于指示目录，因此为了稍作区别，我们使用“_”。）第一个名字是拥有该仓库的 GitHub 帐户名称；第二个名字则是该单个仓库的名称。将仓库和文件标识符两相组合，即可用于单独标识本档案库中的单个文件。例如，在本档案库内的 GitHub 帐户"rezendi"中，有个仓库"ykarma"。其内，目录"web"下的文件"package.json"可单独标识为 rezendi_ykarma 中的 /web/package.json。
 
-各种类型的文件具有不同用途。 GitHub 档案库包含大量文本文件，这意味着，这些文件中的数据代表书面语言。 大多数软件均采用文本文件进行编写，其中包含高度结构化的文本，称为“源代码”。 一种称为“编译器”的特殊程序能将人类可读的源代码转换为计算机可读的指令，即编译代码或机器代码。
+各种类型的文件具有不同用途。GitHub 档案库包含大量文本文件，这意味着，这些文件中的数据代表书面语言。大多数软件均采用文本文件进行编写，其中包含高度结构化的文本，称为“源代码”。一种称为“编译器”的特殊程序能将人类可读的源代码转换为计算机可读的指令，即编译代码或机器代码。
 
-文本文件以外的文件，如用于显示可视图像或包含编译代码的文件，则通常指代二进制文件。 遗憾的是，此术语具有误导性，毕竟文本文件最终也是由 1 和 0 构成。 为此，我们将文本文件以外的文件称为“非文本文件”。
+文本文件以外的文件，如用于显示可视图像或包含编译代码的文件，则通常指代二进制文件。遗憾的是，此术语具有误导性，毕竟文本文件最终也是由 1 和 0 构成。为此，我们将文本文件以外的文件称为“非文本文件”。
 
-使用 1 和 0 代表人类书面语言的方法为数众多。 由于历史原因，大多数源代码最初是由拉丁语脚本编写而成。 拉丁语脚本具有 26 个基本字符，用于表示可口头表达的单词，每个单词设有两种形式，即大写和小写。 此外，该脚本还有 10 位数可用来表示数字。 拉丁语脚本连同用于指示结构和其他概念的各种其他关联符号，均以 1 和 0 的格式（称为"ASCII"）进行编码。该格式可表示 128 个不同字符，并且由于历史原因，多年以来曾在大多数软件中占主导地位。
+使用 1 和 0 代表人类书面语言的方法为数众多。由于历史原因，大多数源代码最初是由拉丁语脚本编写而成。拉丁语脚本具有 26 个基本字符，用于表示可口头表达的单词，每个单词设有两种形式，即大写和小写。此外，该脚本还有 10 位数可用来表示数字。拉丁语脚本连同用于指示结构和其他概念的各种其他关联符号，均以 1 和 0 的格式（称为"ASCII"）进行编码。该格式可表示 128 个不同字符，并且由于历史原因，多年以来曾在大多数软件中占主导地位。
 
-然而，在人类以书面语言表达自我的众多方式中，拉丁语脚本仅为一个小分支。 为了支持其他脚本，同时允许所有编写目的为使用 ASCII 的软件，在无需更改的情况下继续运行，（此概念称为“向后兼容”），"UTF-8"数据格式得以随之引入。
+然而，在人类以书面语言表达自我的众多方式中，拉丁语脚本仅为一个小分支。为了支持其他脚本，同时允许所有编写目的为使用 ASCII 的软件，在无需更改的情况下继续运行，（此概念称为“向后兼容”），"UTF-8"数据格式得以随之引入。
```

Comment:
The word “script” here means **a kind of writing system**, instead of __a written text of a play__ or __program code written for computers__, so “文字” here shall be used in place of “(语)脚本”.

```suggestion
使用 1 和 0 代表人类书面语言的方法为数众多。由于历史原因，大多数源代码最初是由拉丁文字编写而成。拉丁文字具有 26 个基本字符，用于表示可口头表达的单词，每个单词设有两种形式，即大写和小写。此外，该文字还有 10 位数可用来表示数字。拉丁语文字连同用于指示结构和其他概念的各种其他关联符号，均以 1 和 0 的格式（称为"ASCII"）进行编码。该格式可表示 128 个不同字符，并且由于历史原因，多年以来曾在大多数软件中占主导地位。

然而，在人类以书面语言表达自我的众多方式中，拉丁文字仅为一个小分支。 为了支持其他文字，同时允许所有编写目的为使用 ASCII 的软件，在无需更改的情况下继续运行，（此概念称为“向后兼容”），"UTF-8"数据格式得以随之引入。
```

Reply:
Thanks! Fixed in 6d8de46.